tests/ert/ui_tests/cli/test_update.py

```diff
@@ -150,7 +144,6 @@ def test_update_lowers_generalized_variance_or_deactives_observations(
     )
     num_groups = data.draw(st.integers(min_value=1, max_value=num_points))
     per_group = num_points // num_groups
-    print(num_groups, num_points, per_group)
```

Comment:
use the opportunity to correct typo in test name, deactives -> deactivates ?

Reply:
Fixed